tests/core/test_dataset.py

```diff
@@ -288,6 +288,15 @@ def test_array_finalize__multiple_slicing(self):
             else:
                 self.assertTrue(np.allclose(_new_range, self._dset["ranges"][_dim]))
 
+    def test_array_finalize__reordering(self):
+        obj = self.get_random_dataset(1)
+        _slicer = np.arange(obj.shape[0] - 1, -1, -1)
+        _new = obj[_slicer]
+        self.assertEqual(_new.axis_labels, obj.axis_labels)
+        self.assertEqual(_new.axis_units, obj.axis_units)
+        self.assertTrue(np.allclose(_new.axis_ranges[0], obj.axis_ranges[0][_slicer]))
+        self.assertEqual(obj.shape, _new.shape)
+
     def test_array_finalize__insert_data_simple(self):
```

Comment:
I am just checking very briefly. Is here some kind of assert missing?

Reply:
Good catch, that must have slipped somehow